component-library/component-lib/src/lib/form-components/input/input.component.ts

```diff
@@ -145,11 +145,9 @@ export class InputComponent implements ControlValueAccessor, OnInit {
 
     //set disable to true when form is disabled
     this.config.formGroup.valueChanges.subscribe((change) => {
-      if (change[this.config.id] === undefined) {
-        this.disabled = true;
-      } else {
-        this.disabled = false;
-      }
+      this.disabled = <boolean>(
```

Comment:
Can you use the 'as' syntax? It's used elsewhere and I'd rather keep it consistent.